lua/presence/init.lua

```diff
@@ -459,12 +461,10 @@ end
 
 -- Update Rich Presence for the provided vim buffer
 function Presence:update_for_buffer(buffer, should_debounce)
-    if should_debounce and self.last_activity.file == buffer then
-        self.log:debug(string.format("Activity already set for %s, skipping...", buffer))
-        return
-    end
 
     local activity_set_at = os.time()
+    -- If we shouldn't debounce and we trigger an activity, keep this value the same. Otherwise set it to the current time.
```

Comment:
Looks like I'm getting a luacheck error here since the line is too long:
```
Checking lua/lib/log.lua                          OK
Checking lua/presence/discord.lua                 OK
Checking lua/presence/file_assets.lua             OK
Checking lua/presence/file_explorers.lua          OK
Checking lua/presence/init.lua                    1 warning

    lua/presence/init.lua:468:121: line is too long (123 > 120)

Checking lua/presence/plugin_managers.lua         OK

Total: 1 warning / 0 errors in 6 files
```
We can just move the latter sentence to the next line:
```suggestion
    -- If we shouldn't debounce and we trigger an activity, keep this value the same.
    -- Otherwise set it to the current time.
```